stub/stub.go

```diff
@@ -8,13 +8,20 @@ package stub
 
 import (
 	"bytes"
+	"flag"
 	"fmt"
 	"io"
+	"io/ioutil"
 	"log"
+	"os"
+	"path/filepath"
 	"runtime"
+	"runtime/pprof"
 	"sync/atomic"
 )
 
+var serverprofile = flag.String("serverprofile", "", "This is for specifying the prefix of the output file for the profile")
```

Comment:
That's how Go is written. There's all kinds of packages I use that define their own flags.